### PR TITLE
Add support for HTTP POST method as well

### PR DESCRIPTION
**Pull request by papaiatis**
Hi,

I've added support for POST method as well. Please review it and include it in your version.
Feel free to contact me in case of any questions.

Best regards,
Attila

### CODE REVIEW

**Comment by begriffs** [comment]
Just noticed this PR! So sorry that I did not respond earlier, I'm grateful for your help. I've got some suggestions for parts of your change and then let's get this merged in!!

File: src/paginate-anything.js [hunk 1]
```diff
@@ -55,6 +55,8 @@
           size: '=?',
           passive: '@',
           transformResponse: '=?',
+		  method: '@',
+		  postData: '=?',
```
Comment (begriffs) [inline src/paginate-anything.js]: These lines should get a little more indentation to match the preceding ones.

File: src/paginate-anything.js [hunk 2]
```diff
@@ -129,9 +131,10 @@
           function requestRange(request) {
             $scope.$emit('pagination:loadStart', request);
             $http({
-              method: 'GET',
+              method: $scope.method != null ? $scope.method : 'GET',
```
Comment (begriffs) [inline src/paginate-anything.js]: What about `method: $scope.method || 'GET'` ?

File: src/paginate-anything.js [hunk 3]
```diff
@@ -129,9 +131,10 @@
...
               url: $scope.url,
               params: $scope.urlParams,
+			  data: $scope.postData,
```
Comment (begriffs) [inline src/paginate-anything.js]: More indentation for this line as well.

**Comment by begriffs** [comment]
Also can you update the readme as well please to mention these new parameters"

**Comment by begriffs** [comment]
If you fix up the items I mentioned above then I'll be happy to merge this pull request!

**Comment by papaiatis** [comment]
Sorry. It's been a busy month. I'll spare some time today to fix those
issues. Thanks!

**Comment by begriffs** [comment]
Still happy to get this merged once you make the fixes. Thanks Attila!

**Comment by begriffs** [comment]
:tada: Yay, it's merged! Thank you. :balloon:

**Pull request merged**